src/main/java/hw_1/Task_1.java

```diff
@@ -0,0 +1,14 @@
+package hw_1;
+
+public class Task_1 {
+    public static void main(String[] args)
+    {
+        String name = "Antonina";
+        int age = 23;
+        float weight = 50.5F;
+
+
```

Comment:
You have a 3 extra spaces above. Please leave only one

Reply:
fixed